Source/Core/Core/NetPlayClient.cpp

```diff
@@ -57,6 +57,10 @@ NetPlayClient::~NetPlayClient()
 
   if (m_is_connected)
   {
+    m_should_compute_MD5 = false;
+    m_dialog->AbortMD5();
+    if (m_MD5_thread.joinable())
```

Comment:
Doesn't .detach() set joinable to false, so this will never be joined?

Reply:
Yeah you're right, no point in joining the thread. It's also now randomly segfaulting in another place, so needs more null checks.

Reply:
This has been addressed by handling the thread more sanely.